### DISCUSSION TITLE
openapi的位置

### DISCUSSION THREAD

**Discussion by haoshen0611** [kubeapps/kubeapps#4608] [General]
请问在哪里找到之前版本的openapi，我知道在最新版本中位置是[kubeapps/dashboard/public/openapi.yaml，但是之前版本的api找不到呢？

**Comment by antgamdia** [comment]
Hi! We don't actively keep the previous versions of the openapi.yaml file, however, you can always explore the repo at any version tag to find the version used by that version.

For instance, for the v2.4.0: https://github.com/vmware-tanzu/kubeapps/blob/v2.4.0/dashboard/public/openapi.yaml

Hope it helps!
